### DISCUSSION TITLE
[update] Map and list of players

### DISCUSSION THREAD

**Discussion by KuroRaiken** [aXenDeveloper/ips-app-advanced-serverlist#39] [Q&A] [answered]
1. Map
Can you provide it separately on the main monitoring with the addition of a server map image, with an off / on button for this function?
2. players
Can you add a list of players who play on the server when we click on it in monitoring?
As implemented on the device (gs serverlist)

**Comment by KuroRaiken** [comment]
screenshots added
Maps
![FPS](https://user-images.githubusercontent.com/103626579/163715199-0e3c89f8-2238-4ef2-a78b-a9d8a6aa7bcb.png)
Players
![STATS](https://user-images.githubusercontent.com/103626579/163715215-4bda93ab-c266-45f3-9c13-45342cc434ac.png)

It will be necessary to modify this system than in GS Server List

**Comment by aXenDeveloper** [comment, marked as answer]
1. I don't have plan for this function. Certainly not in version 2.0.0,
2. No, showing the number of players on the servers is a heavy load on the server. I'll never be implemented.

**Reply by KuroRaiken** [reply]
> 1. У меня нет плана для этой функции. Точно не в версии 2.0.0,
Will it be implemented in the future?

**Reply by aXenDeveloper** [reply]
Next week.

**Reply by KuroRaiken** [reply]
> На следующей неделе.

Thank you for this)

**Comment by KuroRaiken** [comment]
We are waiting for maps with a map image with the addition of the display settings for the maps themselves for each server in their folder, so as not to be taken from other resources)

**Reply by KuroRaiken** [reply]
@aXenDeveloper There is news a week has passed already)

**Reply by aXenDeveloper** [reply]
And?

**Reply by KuroRaiken** [reply]
> И?

Version 2.1 will this feature be implemented?

**Reply by aXenDeveloper** [reply]
I don't know.
